app/assets/javascripts/modules/track-click.js

```diff
@@ -5,24 +5,36 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   Modules.TrackClick = function () {
     this.start = function (element) {
-      element.on('click', trackClick);
+      var trackable = '[data-track-category][data-track-action]';
 
-      var options = {transport: 'beacon'},
-          category = element.attr('data-track-category'),
-          action = element.attr('data-track-action'),
-          label = element.attr('data-track-label'),
-          dimension = element.attr('data-track-dimension'),
-          dimensionIndex = element.attr('data-track-dimension-index');
-
-      if (label) {
-        options.label = label;
+      if (element.is(trackable)) {
+        element.on('click', trackClick);
+      } else {
+        element.on('click', trackable, trackClick);
       }
```

Comment:
I'm not sure how sophisticated jQuery's event delegation is however if it works as I think it does then using `.find` will add a listener for each element, rather than one listener.

There might be a way we can delegate all links however which would make the code simpler to understand and be performant.